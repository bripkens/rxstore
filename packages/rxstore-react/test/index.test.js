/* eslint-env jest */

test('placeholder', () => {});
