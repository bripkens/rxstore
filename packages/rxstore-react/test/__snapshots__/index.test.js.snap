// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`connectTo must support recompose style hoc 1`] = `
<div
  count={0}
/>
`;

exports[`connectTo must support recompose style hoc 2`] = `
<div
  count={3}
/>
`;

exports[`connectTo property depending 1`] = `
<div
  count={
    Array [
      5,
      0,
    ]
  }
  n={5}
/>
`;

exports[`connectTo property depending 2`] = `
<div
  count={
    Array [
      5,
      0,
      3,
    ]
  }
  n={5}
/>
`;

exports[`connectTo property updates must recalculate observables 1`] = `
<div
  count={
    Array [
      5,
      0,
    ]
  }
  n={5}
/>
`;

exports[`connectTo property updates must recalculate observables 2`] = `
<div
  count={
    Array [
      3,
      0,
    ]
  }
  n={3}
/>
`;

exports[`connectTo static connect 1`] = `
<div
  count={0}
/>
`;

exports[`connectTo static connect 2`] = `
<div
  count={3}
/>
`;
